releases/v16.rst

```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 16.0 Release
+#######################
+
+Version 16.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
+in the :ref:`extensive release notes <release-v16-0>`.  We have also
```

Comment:
Looks like you're using some weird hard-wrapping here. Make sure you use either one-sentence per line or one-paragraph per line line formatting for `pipelines_lsst_io`.

Reply:
Here I let the original wrapping, I just updated from 15.0 to 16.0...